sw/airborne/modules/computer_vision/lib/vision/act_fast.c

```diff
@@ -22,16 +22,18 @@ All rights reserved.
  * de Croon, G.C.H.E. "ACT-FAST: efficiently finding corners by actively exploring images.", in submission.
  *
  */
-
 #include "fast_rosten.h"
 #include "act_fast.h"
 #include "math.h"
 #include "image.h"
 #include "../../opticflow/opticflow_calculator.h"
 
+// ACT-FAST agents arrays
 // equal to the maximal number of corners defined by fast9_rsize in opticflow_calculator.c
+// TODO Currently hardcoded to two cameras
 #define MAX_AGENTS FAST9_MAX_CORNERS
-struct agent_t agents[MAX_AGENTS];
+struct agent_t agents_arr[MAX_AGENTS];
+struct agent_t *agents[2] = {agents_arr, agents_arr + FAST9_MAX_CORNERS / 2};
```

Comment:
Would it make more sense to define the max number of agents per camera? That way, adding a second camera later does not change tracking performance on the first. I.e.:
```
struct agent_t agents_arr[2*MAX_AGENTS];
struct agent_t *agents[2] = {agents_arr, agents_arr + MAX_AGENTS};
```

Also, I suggest using MAX_AGENTS/2 over FAST9_MAX_CORNERS/2 for readability.

Possible shorthand:
```
struct agent_t agents[2][MAX_AGENTS];
```

Reply:
I implemented your suggestion